src/domains/nodo-common/07_sftp.tf

```diff
@@ -22,6 +22,17 @@ module "sftp" {
   access_tier                   = "Hot"
   is_hns_enabled                = true
 
+  blob_versioning_enabled = false
+
+  #  blob_delete_retention_days = var.sftp_sa_delete_retention_days
+  #  blob_change_feed_enabled = var.enable_sftp_backup
+  #  blob_change_feed_retention_in_days = var.enable_sftp_backup ? var.sftp_sa_backup_retention_days : null
+  #  blob_container_delete_retention_days =  var.sftp_sa_backup_retention_days
+  #  blob_storage_policy ={
+  #    enable_immutability_policy = false
+  #    blob_restore_policy_days = var.sftp_sa_backup_retention_days
+  #  }
```

Comment:
this code can be deleted?

Reply:
i left it there because at the momemnt, with the V2 provider, it can't be used. but it must be used once the v3 migration is done